libraries/lib-datahandler/src/main/thrift/licenses.thrift

```diff
@@ -84,12 +84,10 @@ struct License {
     300: optional map<string, string> additionalData,
 
     // Additional informations
-	// 10: optional bool GPLv2Compat,
-	// 11: optional bool GPLv3Compat,
 	12: optional string reviewdate,
 
-	15: optional Ternary GPLv2Compat = sw360.Ternary.UNDEFINED,
```

Comment:
pls. let's check if a migration would be required and also ... it is recommended to use different numbers (for new fields) even if already used are not used anymore. so proposal would be to use 17 and 18 as numbering.

Reply:
Pardon for my dummy question, about 'migration', do you mean that I should prepare script for migration similar to theses script:

https://github.com/eclipse/sw360/tree/master/scripts/migrations

Reply:
I've pushed new commit:
- Refine enum numbering
- Add migration script

Could you please review?